packages/cra-template-typescript/template/src/modules/network/services/reducers/index.ts

```diff
@@ -0,0 +1,21 @@
+import { createReducer } from '@reduxjs/toolkit';
+import { ConnectionStatus } from '../../constants';
+import getConnectionStatus from '../utils/getConnectionStatus';
+
+import * as actions from '../actions';
+
+const initialState = {
+    status: getConnectionStatus(),
+};
+
+const networkReducer = createReducer(initialState, builder => {
```

Comment:
💡 Maybe you could have used `createSlice` to create actions and reducer at once.

Reply:
Good point, I'll check it out